portfolio/pom.xml

```diff
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.8.6</version>
```

Comment:
Indentation

Reply:
I am seeing the correct indentation on Cloud Shell but for some reason it changes once pushed
![Screenshot 2020-06-02 at 1 15 30 PM](https://user-images.githubusercontent.com/32528244/83550516-f6de3080-a4d4-11ea-864c-710870766a12.png)